packages/suite-desktop-core/e2e/playwright.config.ts

```
@@ -34,7 +36,7 @@ const config: PlaywrightTestConfig = {
     reporter: process.env.GITHUB_ACTION
         ? [['list'], ['@currents/playwright'], ['html', { open: 'never' }]]
         : [['list'], ['html', { open: 'never' }]],
-    timeout: 1000 * 60 * 5,
+    timeout: process.env.GITHUB_ACTION ? timeoutCIRun : timeoutLocalRun,
```

We want locally tests to fail fast during troubleshooting.